components/nexus-repository/src/test/java/org/sonatype/nexus/repository/storage/StorageFacetImplIT.java

```diff
@@ -403,6 +403,12 @@ public void roundTripTest() {
       tx.deleteAsset(tx.findAsset(asset1Id, bucket));
       tx.deleteComponent(tx.findComponent(componentId, bucket));
 
+      tx.commit();
```

Comment:
Since browse now runs async on a separate thread, it doesn't see inflight changes not yet committed in the current transaction. So if you want changes to appear when browsing, as in this IT, the current changes must be committed. 

PS. transactions are re-usable, so just adding:
```
tx.commit();
tx.begin();
```
should also work, no need to close out the old transaction and open a new one.